charts/dbildungscloud-iam-keycloak/templates/deployment.yaml

```diff
@@ -16,15 +16,15 @@ spec:
     spec:
       containers:
         - name: keycloak
-          image: "{{ .Values.image.repository }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.appVersion }}"
```

Comment:
Despite being defined as `appVersion` in Chart.yaml it has to be referenced as `.Chart.AppVersion`.